sdk/python/feast/loaders/ingest.py

```diff
@@ -41,8 +43,11 @@ def _encode_pa_tables(file: str, fs: FeatureSet, row_group_idx: int) -> List[byt
             File directory of all the parquet file to encode.
             Parquet file must have more than one row group.
 
-        fs (feast.feature_set.FeatureSet):
-            FeatureSet describing parquet files.
+        feature_set (str):
+            FeatureSet describing the full feature set refference in the format f"{project}/{name}:{version}".
```

Comment:
Typo: "reference"

The type isn't `FeatureSet` anymore so I would drop the "FeatureSet describing the full…", keep the rest after that.